class_heap.c

```diff
@@ -0,0 +1,61 @@
+#include "class_heap.h"
+
+#define MAX_HEAP_SIZE 100
```

Comment:
Why `100` here?

Reply:
I think 100 is enough in most cases. Or should I implement a dynamic heap to handle this?

Reply:
It is fine at present. You can add some comments starting with "FIXME" or "TODO" to address.